massgov/mayflower-patternlab
# toggle js-location-listing class at layout breakpoint

## Description
Disable the scroll up to map on mobile on location listing- this feels very jarring, and also does not add any additional info.
## Related Issue / Ticket

- [JIRA issue: DP-5525](https://jira.state.ma.us/browse/DP-5525)

## Steps to Test

- Before: [Mayflower location listing](http://mayflower.digital.mass.gov/?p=pages-location-listing)
- After: [UAT location listing](https://clairesunstudio.github.io/mayflower/?p=pages-location-listing)
1. resize window to 910px breakpoint
2. for window size > 910px, not change
3. for window size < 911px, click/hover on any location listing, check the following:
 - [ ] disable auto scroll to the map
 - [ ] disable active state
 - [ ] disable map update

## Screenshots
![location-listing](https://user-images.githubusercontent.com/5789411/31683989-78d476e4-b34c-11e7-9409-c1edfaf3cefa.gif)


## Additional Notes:
#### Help needed:
when resize down to window size < 911px, map doesn't recenter.

#### Impacted Areas in Application


* N/A

#### @TODO


* N/A

#### Today I learned...
